pxf/pxf-api/src/main/java/org/apache/hawq/pxf/api/utilities/Utilities.java

```diff
@@ -209,7 +209,8 @@ public static boolean useAggBridge(InputData inputData) {
             LOG.error("Unable to load accessor class: " + e.getMessage());
             return false;
         }
-        return (inputData != null) && (inputData.getAggType() != null)
+        return (inputData != null) && !inputData.hasFilter()
```

Comment:
Add an additional check to filter out queries that have aggregate that is not count(*)